internal/deviations/deviations.go

```diff
@@ -1691,3 +1691,18 @@ func SyslogOCUnsupported(dut *ondatra.DUTDevice) bool {
 func TransceiverConfigEnableUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetTransceiverConfigEnableUnsupported()
 }
+
+// AFTSummaryOCUnsupported returns true if the device does not support aft summary oc path.
+func AFTSummaryOCUnsupported(dut *ondatra.DUTDevice) bool {
+	return lookupDUTDeviations(dut).GetAftSummaryOcUnsupported()
+}
+
+// ISISLSPTlvsOCUnsupported returns true if the device does not support isis lsp tlvs.
+func ISISLSPTlvsOCUnsupported(dut *ondatra.DUTDevice) bool {
+	return lookupDUTDeviations(dut).GetIsisLspTlvsOcUnsupported()
+}
+
+// ISISAdjacencyStreamUnsupported returns true if the device does not support isis adjacency with STREAM telemetry.
+func ISISAdjacencyStreamUnsupported(dut *ondatra.DUTDevice) bool {
```

Comment:
Add a relevant OC path/subtree reference here as a comment.     Does this refer to the subtree `/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/extended-is-reachability/neighbors/neighbor/instances/instance/subtlvs/subtlv/adjacency-sids`?